packages/app-bridge/src/tests/ColorDummy.ts

```diff
@@ -6,108 +6,114 @@ export class ColorDummy {
     static red(id = 767): Color {
         return {
             id,
-            name: 'Red',
             sort: 1,
-            nameCss: 'red',
-            hex: 'ff0000',
-            red: 255,
-            green: 0,
-            blue: 0,
-            alpha: 255,
-            hue: 361,
-            saturation: 100,
-            lightness: 46,
-            c: 0,
-            m: 100,
-            y: 100,
-            k: 0,
-            pantone: null,
-            ral: null,
-            oracal: null,
-            pantoneCoated: null,
-            pantoneUncoated: null,
-            cmykCoated: null,
-            cmykUncoated: null,
-            cmykNewspaper: null,
-            ncs: null,
-            pantoneCp: null,
-            pantonePlastics: null,
-            pantoneTextile: null,
-            hks: null,
-            threeM: null,
-            lab: null,
+            title: 'Red',
+            revision: {
+                nameCss: 'red',
+                hex: 'ff0000',
+                red: 255,
+                green: 0,
+                blue: 0,
+                alpha: 255,
+                hue: 361,
+                saturation: 100,
+                lightness: 46,
+                c: 0,
+                m: 100,
+                y: 100,
+                k: 0,
+                pantone: null,
+                ral: null,
+                oracal: null,
+                pantoneCoated: null,
+                pantoneUncoated: null,
+                cmykCoated: null,
+                cmykUncoated: null,
+                cmykNewspaper: null,
+                ncs: null,
+                pantoneCp: null,
+                pantonePlastics: null,
+                pantoneTextile: null,
+                hks: null,
+                threeM: null,
+                lab: null,
+            },
         };
     }
 
     static yellow(id = 768): Color {
         return {
             id,
-            name: 'Yellow',
             sort: 2,
-            nameCss: 'yellow',
-            red: 255,
-            green: 255,
-            blue: 0,
-            hex: 'ffff00',
-            alpha: 255,
-            hue: 361,
-            saturation: 100,
-            lightness: 46,
-            c: 0,
-            m: 0,
-            y: 100,
-            k: 0,
-            pantone: null,
-            ral: null,
-            oracal: null,
-            pantoneCoated: null,
-            pantoneUncoated: null,
-            cmykCoated: null,
-            cmykUncoated: null,
-            cmykNewspaper: null,
-            ncs: null,
-            pantoneCp: null,
-            pantonePlastics: null,
-            pantoneTextile: null,
-            hks: null,
-            threeM: null,
-            lab: null,
+            title: 'Yellow',
+            revision: {
```

Comment:
should this now be adjusted to reflect the new color type encapsulation?

Reply:
yes adjusted